### PR TITLE
Improve tooltip for diff selection

### PR DESCRIPTION
**Pull request by gerhardol**
## Proposed changes

Align tooltip with changes in #7899
Suggest inclusion in 3.4, even if translation is updated. (CThis is a clarification, not breaking existing translations)

### Before

image

### After

image

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by gerhardol** [comment]
@msftbot merge in 48 hours

File: GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.cs [hunk 1]
```diff
@@ -10,9 +10,12 @@ public partial class DiffViewerSettingsPage : SettingsPageWithHeader
 
 - For a single selected commit, show the difference with its parent commit.
 - For a single selected merge commit, show the difference with all parents.
-- For two selected commits, show the difference between the commits as well as the difference between a common ancestor (BASE) to the last selected commit.
+- For two selected commits with a common ancestor (BASE):
+ - Show the difference between the commits.
+ - The difference of unique files from BASE to each of the selected commits.
+ - The difference of common files (identical changes) from BASE to the commits.
```
Comment (drewnoakes) [inline GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.cs:16]: A little more indentation would make this nesting more visually obvious, especially in a non-monospace font. I didn't see it at first in the screenshot.

```suggestion
- For two selected commits with a common ancestor (BASE):
   - Show the difference between the commits.
   - The difference of unique files from BASE to each of the selected commits.
   - The difference of common files (identical changes) from BASE to the commits.
```

File: GitUI/Translation/English.xlf [hunk 2]
```diff
@@ -1149,9 +1149,12 @@ Please make sure git (Git for Windows or cygwin) is installed or set the correct
 
 - For a single selected commit, show the difference with its parent commit.
 - For a single selected merge commit, show the difference with all parents.
-- For two selected commits, show the difference between the commits as well as the difference between a common ancestor (BASE) to the last selected commit.
+- For two selected commits with a common ancestor (BASE):
+ - Show the difference between the commits.
+ - The difference of unique files from BASE to each of the selected commits.
+ - The difference of common files (identical changes) from BASE to the commits.
```
Comment (drewnoakes) [inline GitUI/Translation/English.xlf:1155]:
```suggestion
- For two selected commits with a common ancestor (BASE):
   - Show the difference between the commits.
   - The difference of unique files from BASE to each of the selected commits.
   - The difference of common files (identical changes) from BASE to the commits.
```

**Comment by RussKie** [comment]
We can include it in v3.4

File: GitUI/Translation/English.xlf (hunk 2 above)
Comment (gerhardol) [reply GitUI/Translation/English.xlf:1155]: Adding two spaces

**Comment by gerhardol** [comment]
@msftbot merge in 24 hours

**Pull request merged**